### DISCUSSION TITLE
How to use multiple docker repository in one config

### DISCUSSION THREAD

**Discussion by przemekgorzynski** [renovatebot/renovate#29229] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

I have following custom yaml (ansible inventory) with defined docker images:

```
  nginx_name: docker.io/nginx
  nginx_tag: 1.25.0
  app_image_name: private_repo.io/private_app
  app_tag: 0.14.0
```

In renovate config have two regexp managers to check for dependencies based on repo name

```
  "customManagers": [
    {
      "customType": "regex",
      "description": "docker.io",
      "fileMatch": ["^inventory/.*.yml$"],
      "matchStrings": [
        ".*docker.io/(?<depName>\\S+)\\n\\s+.*nginx_tag:\\s+(?<currentValue>\\S+)"
      ],
      "datasourceTemplate": "docker",
      "versioningTemplate": "docker"
    },
    {
      "customType": "regex",
      "description": "private_repo.io",
      "fileMatch": ["^inventory/.*.yml$"],
      "matchStrings": [
        ".*private_repo.io/(?<depName>\\S+)\\n\\s+.*app_tag:\\s+(?<currentValue>\\S+)"
      ],
      "datasourceTemplate": "docker",
      "versioningTemplate": "docker"
    }
  ]
```

In detected dependencies renovate detects:

```
inventory/inv.yml
nginx 1.25.0

inventory/inv.yml
private_app 0.14.0
```

1st regex configuration works fine and it checks against dockerhub repository. 
However 2nd regex gives error `Failed to look up docker package private_app` - renovate tries to lookup in docker hub where image does not exists.
How to reconfigure 2nd regex to check in private repo and overwrite dependency from  `private_app` to `private_repo.io/private_app` - checked and if dependency is defined as `private_repo.io/private_app` it works.

If I add to 2nd regexp `"registryUrlTemplate": "private_repo.io/",` it returns 

> Renovate tried to run on this repository, but found these problems.
> WARN: Invalid regex manager registryUrl
> WARN: Package lookup failures

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Try adding a https:// prefix to the registry URL
